coalib/bears/requirements/NpmRequirement.py

```diff
@@ -49,7 +49,13 @@ def is_installed(self):
 
         :param return: True if dependency is installed, false otherwise.
         """
-        cmd = ['npm', 'show', self.package]
-        if platform.system() == 'Windows':  # pragma: no cover
-            cmd = ['cmd', '/c'] + cmd
-        return not call_without_output(cmd)
+        for cmd in (['npm', 'list', self.package],
+                    ['npm', 'list', '-g', self.package]):
+
+            if platform.system() == 'Windows':  # pragma: no cover
+                cmd = ['cmd', '/c'] + cmd
```

Comment:
don't we have a function somewhere that does this windows stuff for you? Shell.py? CC @Makman2